### ISSUE TITLE
A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader : The controls such as "Repository" and "License" are not announced with proper role for screen reader users

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#104552]
“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11yMAS; #A11yTCS; #MAC; #ColorContrast; #A11y_VSCode; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#FTP;#VoiceOver;#MAS1.3.1;#MAS4.1.2;
Environment Details:
Application Name: Azure Extensions for VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

**Repro Steps:**
Open Visual studio code
Click on extension button and search for "Azure Functions"
Click on install and move to the "Azure" button, the "Default Screen" will appear.
Start screen reader and navigate to controls such as "Repository" and "License" and observe if these are announced with role as link or not.
**Actual:**
When screen reader users navigate to the controls such as "Repository" and "License" they are announced as group and text which is incorrect as on activating them web page appear on the screen so they should be announced with proper role for screen reader users.

**Expected:**
The controls such as "Repository" and "License" should be announced as 'link' for screen reader users when they navigate to these controls.

**Attachment link for Reference:**
[104552_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader.The.controls.such.as.Repository.and.License.are.not.announced.with.proper.role.for.screen.reader.users.zip](https://github.com/microsoft/vscode/files/5068686/104552_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader.The.controls.such.as.Repository.and.License.are.not.announced.with.proper.role.for.screen.reader.users.zip)

**Comment by isidorn** [comment]
Please file against ms-azuretools.vscode-azurefunctions extension.
Then that team can identify if it is really an issue in vscode and file it here with minimal human readable steps.

**Comment by nirmalsahoo** [comment]
@isidorn We have already logged bug in ms-azuretools.vscode-azurefunctions extension and their team says that this issue is external to VSCode then we have filed issue .

Bug reference: https://github.com/microsoft/vscode-azurefunctions/issues/2281
Please reactivate this issue.

**Comment by isidorn** [comment]
Makes sesne but that team should then file an issue in VS Code. They should put effort in filling an issue and use VS Code language when filling an issue, not just forward you to file an issue in our repository. They should search for duplicates etc..
